{"url":"https://api.github.com/repos/owncloud/core/issues/2376","labels_url":"https://api.github.com/repos/owncloud/core/issues/2376/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/2376/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/2376/events","html_url":"https://github.com/owncloud/core/issues/2376","id":12095047,"number":2376,"title":"lost files through sync","user":{"login":"philippsalvisberg","id":3883171,"avatar_url":"https://gravatar.com/avatar/07862145889e6a1a3d39865c89ce5965?d=https%3a%2f%2fidenticons.github.com%2f9f5a1e0eca03645c2ede312fd0061839.png&r=x","gravatar_id":"07862145889e6a1a3d39865c89ce5965","url":"https://api.github.com/users/philippsalvisberg","html_url":"https://github.com/philippsalvisberg","followers_url":"https://api.github.com/users/philippsalvisberg/followers","following_url":"https://api.github.com/users/philippsalvisberg/following{/other_user}","gists_url":"https://api.github.com/users/philippsalvisberg/gists{/gist_id}","starred_url":"https://api.github.com/users/philippsalvisberg/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/philippsalvisberg/subscriptions","organizations_url":"https://api.github.com/users/philippsalvisberg/orgs","repos_url":"https://api.github.com/users/philippsalvisberg/repos","events_url":"https://api.github.com/users/philippsalvisberg/events{/privacy}","received_events_url":"https://api.github.com/users/philippsalvisberg/received_events","type":"user","site_admin":false},"labels":[],"state":"closed","assignee":null,"milestone":null,"comments":2,"created_at":"2013-03-16t13:42:54z","updated_at":"2013-03-16t17:25:38z","closed_at":"2013-03-16t17:25:38z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"hi, \r\n\r\ni've lost files after synchronization (just in a test scenario - because i had strange behaviors already with 4.5). some of them are in the trash folder, but some (6 out of 1131) are completely vanished.\r\n\r\ni've done the following:\r\n\r\n1) changed php.ini\r\n- memory_limit = 1032m\r\n- post_max_size = 1032m\r\n- upload_max_filesize = 1024m\r\n\r\n2) installed the server 5.0 using default settings and created an additional user\r\n\r\n3) installed and configured mac os client 1.2.1 on machine 1\r\n\r\n4) installed and configured mac os client 1.2.1 on machine 2 (same local and remote path as machine 1)\r\n\r\n5) copied 3 folder containing 1131 files, total 2.5 gb, largest file 245 mb. sync started immediately on client 1 and client 2. at the end i had the following result (performance was ok)\r\n\r\n- local directory on machine 1 contained 1125 files\r\n- remote directory on server (data/<user>/files/clientsync contained 1080 files\r\n- trash directory on server (data/<user>/files_trashbin/):\r\n  - ./files : 41 files\r\n  - ./keyfiles: 0 files\r\n  - ./versions: 22 files\r\n  - plus 4 deleted folders \r\n- local directory on machine 2 contained 1066 files (including the largest one)\r\n\r\nis there something i could do to avoid lost files?\r\n\r\nthanks","closed_by":{"login":"mtgap","id":1097106,"avatar_url":"https://gravatar.com/avatar/454a73ee5feeb428d0c163a48903332c?d=https%3a%2f%2fidenticons.github.com%2f52d29a66f285a1d61125dd69ac0766b2.png&r=x","gravatar_id":"454a73ee5feeb428d0c163a48903332c","url":"https://api.github.com/users/mtgap","html_url":"https://github.com/mtgap","followers_url":"https://api.github.com/users/mtgap/followers","following_url":"https://api.github.com/users/mtgap/following{/other_user}","gists_url":"https://api.github.com/users/mtgap/gists{/gist_id}","starred_url":"https://api.github.com/users/mtgap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mtgap/subscriptions","organizations_url":"https://api.github.com/users/mtgap/orgs","repos_url":"https://api.github.com/users/mtgap/repos","events_url":"https://api.github.com/users/mtgap/events{/privacy}","received_events_url":"https://api.github.com/users/mtgap/received_events","type":"user","site_admin":false}}